ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/PrivGetFilterChanges.java

```diff
@@ -53,8 +54,9 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
     final String privacyGroupId = requestContext.getRequiredParameter(0, String.class);
     final String filterId = requestContext.getRequiredParameter(1, String.class);
 
-    checkIfPrivacyGroupMatchesAuthenticatedPrivacyUserId(requestContext, privacyGroupId);
-
+    if (privacyController instanceof MultiTenancyPrivacyController) {
```

Comment:
This seems a bit risky. If e.g. someone where to implement multi-tenancy via plugin, or just a different class, this case here could easily be missed leading private data exposure.

Could we unconditionally call `checkIfPrivacyGroupMatchesAuthenticatedPrivacyUserId` and have a method in the privacy controller called? This way it would be more visible to implementers of the privacy controller.

Reply:
Not really fixed, but there is a ticket to get rid of the MultiTanancyPrivacyController!

Reply:
👍